src/scripts/main.js

```diff
@@ -1,3 +1,200 @@
 'use strict';
 
-// write your code here
+const columns = [[], [], [], []];
+const rows = [...document.querySelectorAll('.field-row')];
+const cells = document.querySelectorAll('.field-cell');
+const cellsInRow = rows.map((row) => [...row.querySelectorAll('.field-cell')]);
+const startCellVelues = [2, 2, 2, 2, 2, 2, 2, 2, 2, 4];
+
+const startButton = document.querySelector('.start');
+const startMessage = document.querySelector('.message-start');
+const loseMessage = document.querySelector('.message-lose');
+const winMessage = document.querySelector('.message-win');
+const gameScore = document.querySelector('.game-score');
+
+for (let i = 0; i < 4; i++) {
```

Comment:
Instead of using the nested for loops to fill the columns array, consider using array methods such as 'map' or 'forEach' for better readability and performance.